### DISCUSSION TITLE
Author

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#60] [Idee/Proposte]
Buonasera a tutti/e! Pensate sia opportuno realizzare anche una pagina dedicata ad ogni autore? Potrebbe contenere una foto, una piccola bio, i riferimenti social e i post scritti. Che ne dite?

**Comment by gbvitrano** [comment]
Se non ricordo male nel vecchio sito c'era qualcosa di simile

**Reply by dennisangemi** [reply]
Nel vecchio sito c'era l'immagine e la bio in ogni articolo

**Comment by gbvitrano** [comment]
Ricordo di aver compilato questi campi quando mi sono registrato. ma oggi non c'è traccia dei dati 
![image](https://user-images.githubusercontent.com/20488693/146053727-16d54e26-f19f-4ea3-8919-79de07f5a906.png)
![2021-12-14_18h55_44](https://user-images.githubusercontent.com/20488693/146053811-f0732a87-07e7-4d83-a59a-b9d2f3eb73fb.png)

**Reply by dennisangemi** [reply]
Forse questa potrebbe diventare una issue per @aborruso ?

**Reply by aborruso** [reply]
> Forse questa potrebbe diventare una issue per @aborruso ?

Purtroppo nel file standard di export ci sono soltanto nome, cognome, email e username. La prenderei come occasione per rimettere il tutto in bella.

**Reply by dennisangemi** [reply]
Grazie @aborruso , cosa intendi per "La prenderei come occasione per rimettere il tutto in bella"?

**Comment by dennisangemi** [comment]
@gabacode mi confermi che è possibile realizzare una cosa del genere in Gatsby? È complicato? Ne vale la pena?

**Comment by gbvitrano** [comment]
Con WP già si può fare, ma non trovando i vecchi dati degli autori, non ho più instato il puglin 
![Acquisisci schermata Web_14-12-2021_19043_opendatasicilia it](https://user-images.githubusercontent.com/20488693/146054293-a5e4df19-d613-4de1-9226-6e7f26bce1db.jpeg)

**Reply by gabacode** [reply]
Potremmo e (secondo me) dovremmo usare WP per gestire foto, bio etc così da agganciarli direttamente ai post. Dovremmo provare a vedere se dal vecchio backup riusciamo a recuperarli

**Reply by dennisangemi** [reply]
Se non sbaglio, il vecchio sito sfruttava gravatar. Non ho la minima idea di cosa sia ma ricordo che quando mi sono registrato ho caricato qualcosa lì

**Comment by gabacode** [comment]
> @gabacode mi confermi che è possibile realizzare una cosa del genere in Gatsby? È complicato? Ne vale la pena?

Si, sarebbe utile e carino farlo, secondo me.

**Reply by dennisangemi** [reply]
Grazie! Lo aggiungo alle cose da fare haha

**Comment by gbvitrano** [comment]
@gabacode ho installato il plugin :-) peccato non poter recuperarle tutte
Non dovrebbero essere nel db?
![2021-12-14_21h35_46](https://user-images.githubusercontent.com/20488693/146075499-c488e3de-8c49-4a35-81ed-584cfb681b66.png)

**Reply by gabacode** [reply]
Se c'erano, probabilmente erano nella tabella `wp_options` del vecchio db, in un field chiamato `abh_options` (sarebbe da verificare se questo campo è stato aggiunto all'attuale DB dopo che hai installato il plugin.

Come ti accennavo su Telegram, la tua bio l'ho trovata tramite [wayback machine](https://web.archive.org/web/20201030021217/http://opendatasicilia.it/2017/08/29/palermo-3d-maps-wrld/), se non dovessimo avere un backup del vecchio db forse possiamo provare a fare qualche ricerca e cercare di recuperare il recuperabile.

@aborruso abbiamo per caso un dump del vecchio db?

**Reply by aborruso** [reply]
> @aborruso abbiamo per caso un dump del vecchio db?

Sì, certo. Ma non me ne posso occupare subito. Io - datemi dello scemo - non trovo male che sia vuoto e costringere tutti a riscriverlo.
È roba che si lascia lì e molto spesso c'è la foto di quando si avevano i capelli e il profilo professionale di quando eri il bassista dei Nirvana.

Ma vi farò avere il dump

**Reply by dennisangemi** [reply]
Adesso ho capito cosa intendevi per "rimettere il tutto in bella" ! In effetti non è una cattiva idea haha
